sycamore/functions/document.py

```diff
@@ -8,6 +8,33 @@
 
 
 def split_and_convert_to_image(doc: Document) -> list[Document]:
+    """Split a document into individual pages as images and convert them into Document objects.
+
+    This function takes a Document object, which may represent a multi-page document, and splits it into individual
+    pages. Each page is converted into an image, and a new Document object is created for each page. The resulting
+    list contains these new Document objects, each representing one page of the original document and elements making
+    up the page.
+
+    The input Document object should have a binary_representation attribute containing the binary data of the
+    document (e.g., a PDF).Each page's elements are preserved in the new Document objects, and page-specific properties
```

Comment:
Since this only works for PDFs, I would say "containing the binary data of the pdf document" and drop the e.g..

Reply:
Updated